packages/select/src/SelectInput.tsx

```diff
@@ -16,7 +16,7 @@ export const SelectInput = (StyledSelectInput: any) =>
         focusable={false}
         importantForAccessibility="no"
         placeholder={placeholder}
-        value={value ? value : ''}
+        value={label ? label : ''}
```

Comment:
@jacargentina, Can we add fallback to Value if label is empty?

Reply:
Also take pull from main

Reply:
Done!